src/components/pushClient/Banner.jsx

```diff
@@ -23,24 +24,27 @@ class BannerClient extends Component {
   }
 
   async componentWillMount() {
+    if (Config.banner.isActivated !== true) return
     const seen = (await localforage.getItem(DESKTOP_BANNER)) || false
     if (!seen) {
       const mustSee = !(await isClientAlreadyInstalled())
       if (mustSee) {
-        this.setState(state => ({ ...state, mustShow: true }))
+        this.setState({ mustShow: true })
       }
     }
   }
 
   markAsSeen(element) {
     localforage.setItem(DESKTOP_BANNER, true)
-    this.setState(state => ({ ...state, mustShow: false }))
+    this.setState({ mustShow: false })
     track(element)
   }
 
   render() {
+    if (Config.banner.isActivated !== true) return null
     const { t } = this.props
-    if (!this.state.mustShow) return null
+
+    if (this.state.mustShow !== true) return null
```

Comment:
why do you prefer this `this.state.mustShow !== true`?

Reply:
This modification should not be there